src/main/kotlin/io/bazel/kotlin/plugin/jdeps/k2/JdepsK2Utils.kt

```diff
@@ -28,7 +28,7 @@ internal fun SourceElement.binaryClass(): String? =
   when (this) {
     is KotlinJvmBinarySourceElement -> binaryClass.location
     is JvmPackagePartSource -> this.knownJvmBinaryClass?.location
-    is JavaBinarySourceElement -> this.javaClass.virtualFile.path
+    is VirtualFileBasedSourceElement -> this.virtualFile.path
```

Comment:
This is a breaking change, we might be able to work around this with some less-than-amazing reflection in the short term but it'll be pretty ugly.